sympy/geometry/parabola.py

```diff
@@ -376,6 +376,19 @@ def p_parameter(self):
             p = sign(self.focus.args[1] + y)
         return p * self.focal_length
 
+    @property
+    def latus_rectum(self):
+        """latus rectum of parabola- a special chord that passes through focus
```

Comment:
"""returns the line passing through the focus that is perpendicular to the axis of symmetry"""?

Reply:
yes